DABI.js

```diff
@@ -0,0 +1,321 @@
+    }
+    ZU.processDocuments(ids, scrape);
+}
+
+
+function getSearchResults(doc) {
+    var trs = doc.getElementsByTagName("tr"),
+        tds = null,
+        items = {};
+
+    for (var i = 1; i < trs.length; i++) {
+        tds = trs[i].getElementsByTagName("td");
+        for (var n = 0; n < tds.length; n++) {
+            var url = doc.location.origin + "/cgi-bin/dabi/" + tds[0].firstChild.getAttribute("href"),
+                author = tds[1].innerHTML,
+                title = tds[2].innerHTML.replace(/<br>/g, '. ');
+
+            if (!author == '') {
+                var item = author.replace(/; <br>.*/, ' et al.') + ": " + title;
+            } else {
+                var item = title;
+            }
+            if (!item || !url) continue;
+
+            items[url] = item;
+        }
+    }
+    return items;
+}
+
+function scrape(doc, url) {
+    var newItem = new Zotero.Item('journalArticle');
+    var trs = doc.getElementsByTagName("tr"),
+        headers,
+        contents,
+        items = {};
+
+    //For Loop to populate "items" Object and save tags to an Array.
+    for (var i = 0; i < trs.length; i++) {
+        headers = trs[i].getElementsByTagName("th")[0].textContent;
+        contents = trs[i].getElementsByTagName("td")[0].innerHTML;
+
+        items[headers.replace(/\s+/g, '')] = contents.trim();
+    }
+
+    //set url to fulltext resource, if present; else to database item
+    if (items["URL"] == '') {
+        newItem.url = url;
+    } else {
+        var link = doc.createElement('a');
+        link.innerHTML = items["URL"];
+        newItem.url = link.firstChild.getAttribute("href");
+
+        if (/\.pdf(#.*)?$/.test(newItem.url)) {
+            newItem.attachments = [{
+                url: newItem.url,
+                title: "DABI Full Text PDF",
+                mimeType: "application/pdf"
+            }];
+        }
+    }
+
+
+    //Formatting and saving "title" fields
+    if (items["Titel"]) {
+        newItem.title = items["Titel"].replace(/\*/g, '');
+        var short = newItem.title.replace(/^\W?(?:Die |Der |Das |\.{3}\s?)?/, '');
+        short = short.replace(/\W?(,|:|\?|!|\.|\"|\')/, '').split(' ').slice(0, 6).join(' ');
+        newItem.shortTitle = short.substring(0, 1).toUpperCase() + short.slice(1);
+        if (items["Untertitel"]) {
+            if (/(\?|!|\.)\W?$/.test(newItem.title)) {
+                newItem.title += " " + items["Untertitel"];
+            } else {
+                newItem.title += ": " + items["Untertitel"];
+            }
+        }
+    }
+
+    //Sometimes titles are missing
+    if (items["Untertitel"] && !items["Titel"]) {
+        newItem.title = items["Untertitel"].replace(/\*/g, '');
+        var short = newItem.title.replace(/^\W?(?:Die |Der |Das |\.{3}\s?)?/, '');
+        short = short.replace(/\W?(,|:|\?|!|\.|\"|\').*$/, '').split(' ').slice(0, 6).join(' ');
+        newItem.shortTitle = short.substring(0, 1).toUpperCase() + short.slice(1);
+    }
```

Comment:
Instead of replicating this here, you could just do `if (!items["Titel"]) items["Titel"] = items["Untertitel"];` before the previous if block.

Reply:
Done